src/table/settings/game_settings.ini

```diff
@@ -243,8 +243,20 @@ strval   = STR_CONFIG_SETTING_NONE
 var      = vehicle.never_expire_vehicles
 flags    = SF_NO_NETWORK
 def      = false
+to       = SLV_ENGINE_EXPIRY_OWNED
+
+[SDT_VAR]
+var      = vehicle.engine_expiry
+type     = SLE_UINT8
+from     = SLV_ENGINE_EXPIRY_OWNED
+flags    = SF_GUI_DROPDOWN | SF_NO_NETWORK
+def      = EngineExpiryMode::Off
+min      = EngineExpiryMode::Off
+max      = EngineExpiryMode::Owned
 str      = STR_CONFIG_SETTING_NEVER_EXPIRE_VEHICLES
 strhelp  = STR_CONFIG_SETTING_NEVER_EXPIRE_VEHICLES_HELPTEXT
+strval   = STR_CONFIG_SETTING_NEVER_EXPIRE_VEHICLES_OFF
+cat      = SC_BASIC
```

Comment:
And the hopefully predictable question: why not make it an enum setting, instead of a number? Makes the configuration a bit more self-explaining.